src/components/common/CertifySection.vue

```diff
@@ -65,7 +70,7 @@ export default class CertifySection extends Mixins(DateMixin) {
   @Prop({ default: false }) readonly validate!: boolean
 
   /** To determine whether user input is enabled. */
-  @Prop({ default: false }) readonly disableEdit!: boolean
+  @Prop({ required: true }) readonly disableEdit!: boolean
```

Comment:
I found it confusing with the default, so I made this prop required and pass it in everywhere this component is used.